Dockerfile

```diff
@@ -4,14 +4,18 @@ MAINTAINER Peter Dave Hello <hsu@peterdavehello.org>
 
 RUN echo '@edge http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories && \
     apk -U upgrade && \
-    apk -v add tor@edge curl && \
+    apk -v add shadow tor@edge curl && \
     rm -rf /var/cache/apk/*
-RUN tor --version
+RUN tor --version && mkdir /tor && chown -Rh tor. /tor && chmod -R 700 /tor
```

Comment:
Just curious, why not `chown -Rh tor /tor` but `chown -Rh tor. /tor`?

Reply:
Another question is, is `-h` really needed here? On my all tor instances, I didn't see any symbolic links.